### PR TITLE
Ambari documentation

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build](http://builds.cask.co/browse/CDAP-DOB27-1)

### CODE REVIEW

**Comment by awholegunch** [comment]
[Fails the Quick Build (pre 3.2)](http://builds.cask.co/browse/CDAP-DQBP31-1)...fixing...

**Comment by wolf31o2** [comment]
Passing build link?

**Comment by awholegunch** [comment]
Finally...[passes](http://builds.cask.co/browse/CDAP-DQBP31-5). [Ambari Docs](http://builds.cask.co/artifact/CDAP-DQBP31/shared/build-5/Docs-HTML/3.1.2/en/integrations/ambari/index.html)

File: cdap-docs/integrations/source/ambari/index.rst [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+- **HDFS:** used as the backing file system for distributed storage
+- **MapReduce2:** used for batch operations in workflows and data exploration
+- **YARN:** used for running system services in containers on cluster NodeManagers
+- **HBase:** used for system runtime storage and queues
+- **ZooKeeper:** used for service discovery and leader election
+
+.. rubric:: Optional Dependencies
+
+- **Hive:** used for data exploration using SQL queries via CDAP Explore system service
+- **Spark:** used for running Spark programs within CDAP applications
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: Should we add `(on HDP 2.2+)` to Spark, since it's not available on previous versions?

Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: Should that also be on the 3.0 documentation?

Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: No. It shouldn't. I'll comment over in that PR.

Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: Added.

Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: Thanks, and I was wrong. We already merged `3.0`...

Spark isn't available in CDAP 3.0, so we should comment as such.

**Comment by awholegunch** [comment]
Re-running [Quick Build](http://builds.cask.co/browse/CDAP-DQBP31-6)

**Pull request merged**